operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2053,6 +2086,330 @@ gateway:
         hosts:
           - gateway.gem.example.com
 
+graphite:
+  # -- If true, enables graphite querier and graphite write proxy functionality.
+  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
```

Comment:
@dimitarvdimitrov can you change this URL `.../enterprise-metrics/...`?

Reply:
sure thing https://github.com/grafana/mimir/pull/2830